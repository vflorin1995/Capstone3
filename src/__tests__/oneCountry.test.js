import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import { store } from '../redux/fetch';
import OneCountry from '../modules/oneCountry';

describe('Jest Snapshot testing suite', () => {
  it('OneCountry snapshot', () => {
    const quotes = renderer.create(
      <Provider store={store}>
        <Router>
          <OneCountry />
        </Router>
      </Provider>,
    ).toJSON();
    expect(quotes).toMatchSnapshot();
  });
});
