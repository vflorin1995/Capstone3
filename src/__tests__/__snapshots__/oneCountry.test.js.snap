// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite OneCountry snapshot 1`] = `"Loading"`;
